<launch>
    <node pkg="jarvis_grasp" name="grasp_detector" type="grasp_detector_stub.py"/>
    <node pkg="jarvis_grasp" name="pose_generator" type="PoseGenerator"/>
</launch>
